operational/aws/lambda_functions_with_high_error_rate/lambda_functions_with_high_error_rate.pt

```diff
@@ -1,77 +1,96 @@
-name "AWS Lambda Functions with high error rate"
+name "AWS Lambda Functions With High Error Rate"
 rs_pt_ver 20180301
 type "policy"
-short_description "Report any functions with error rate over parameter. \n See the [README](https://github.com/flexera-public/policy_templates/tree/master/operational/aws/lambda_functions_with_high_error_rate) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+short_description "Report any AWS Lambda functions with high error rates. See the [README](https://github.com/flexera-public/policy_templates/tree/master/operational/aws/lambda_functions_with_high_error_rate) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
 long_description ""
-category "Operational"
 severity "high"
+category "Operational"
 default_frequency "hourly"
 info(
-  version: "4.2.1",
+  version: "5.0.0",
   provider: "AWS",
-  service: "Lambda",
-  policy_set: ""
+  service: "PaaS",
+  policy_set: "Lambda"
```

Comment:
```suggestion
  service: "Lambda",
  policy_set: ""
```

```suggestion
  service: "Lambda",
```
The service name in the bill and API/UI is "Lambda" and I think we should keep that... 

But then that leaves 

here's full list of policy_sets we have now:
https://github.com/flexera-public/policy_templates/blob/9f6bac9366972a232186312580fe77e8c3bbe173/README.md?plain=1#L621

Looking at this list it almost makes me want to make policy_set usually "" unless we can map it to a really good "set".. we have a few but not many

CIS
Rightsize Volumes
Rightsize Storage
...

But we could have policy where we require a review or an approval (via Danger + test for a label added to the PR) to merge a PR where policy_set is something other than ""

Reply:
I disagree, but using `service` for classification of the usage type/resource type has already been done to other policies... Marking this as resolved